From c15ebc1f144efc3810ba1bfa4e4cc4944b0b7cb3 Mon Sep 17 00:00:00 2001
From: Xinzheng Long <xinlon@qti.qualcomm.com>
Date: Thu, 15 Jan 2026 16:11:10 +0800
Subject: [PATCH] Revert "QCLINUX: arm64: configs: qcom.config: Enable
 I2C_QCOM_GENI as built-in"

This reverts commit 24ab2f786dfe65e62c275f0b0748a03f959d3036.

Upstream-Status: Pending
Signed-off-by: Xinzheng Long <xinlon@qti.qualcomm.com>
---
 arch/arm64/configs/qcom.config | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/qcom.config b/arch/arm64/configs/qcom.config
index 4be5a6115b4e..26da044151cf 100644
--- a/arch/arm64/configs/qcom.config
+++ b/arch/arm64/configs/qcom.config
@@ -25,7 +25,6 @@ CONFIG_DMABUF_HEAPS_SYSTEM=y
 CONFIG_EDAC_QCOM=m
 CONFIG_EXPERT=y
 CONFIG_GUNYAH_WATCHDOG=y
-CONFIG_I2C_QCOM_GENI=y
 CONFIG_I6300ESB_WDT=y
 CONFIG_IDLE_INJECT=y
 CONFIG_INPUT_UINPUT=y
